doc(token-api): Add more examples, and a link to terms of use (#1092)
Consensys/doc.linea: docs/api/token-api/overview.mdx

- [Nile](https://www.nile.build/)
- [Moralis](https://moralis.com/)

Please see the [Linea terms of use](https://linea.build/terms-of-service) about third party information.

### Update frequencies
- Token detection and metadata: every two hours
- Historical prices: hourly
- Current prices: every five minutes

## Usage examples

### Filter tokens by security

Getting a raw list of tokens is good.

> But getting a list of tokens that are secure is even better.

Reviewer: ```suggestion
Getting a raw list of tokens is good, but getting a list of tokens that are secure is even better.
```

> You can filter on secure tokens using the `isSecure` query parameter.

Reviewer: ```suggestion
You can filter tokens according to their security status using the `isSecure` query parameter.
```

For a token to be considered as secure,

> it must have a Moralis security score greater than 70 and a market capitalisation greater than $50k.

Reviewer: ```suggestion
it must have a Moralis security score greater than 70 and a market capitalization greater than $50k.
```

```typescript
async function getSecureTokens() {
  const BASE_URL = "https://token-api.linea.build";
  const tokens = await fetch(`${BASE_URL}/tokens?isSecure=true`).then(r => r.json());
  console.log('Secure tokens:', tokens);
}
```

### Get new gems

New gems are tokens that were recently deployed and may be the next big thing.

> Adding a `isSecure` filter might be a good idea to avoid scams.

Reviewer: ```suggestion
Adding an `isSecure` filter might be a good idea to avoid scams.
```

```typescript
async function getNewGems() {
  const BASE_URL = "https://token-api.linea.build";
  const tokens = await fetch(`${BASE_URL}/tokens?order=createdAt&sort=desc`).then(r => r.json());
  console.log('Recent tokens:', tokens);
}
```

> ### Get most swapped tokens

Reviewer: ```suggestion
### Get most-swapped tokens
```

> You can get the most swapped tokens in the last 24h using the `isMostSwapped` query parameter.

Reviewer: ```suggestion
You can get the most-swapped tokens in the last 24 hours using the `isMostSwapped` query parameter.
```
